bin/fetch-historic-measurements.js

```diff
@@ -0,0 +1,289 @@
+/*
+Usage:
+
+1. Setup port forwarding between your local computer and Postgres instance hosted by Fly.io
+  ([docs](https://fly.io/docs/postgres/connecting/connecting-with-flyctl/)). Remember to use a
+  different port if you have a local Postgres server for development!
+   ```sh
+   fly proxy 5454:5432 -a spark-db
+   ```
+
+2. Find spark-db entry in 1Password and get the user and password from the connection string.
+
+3. Run the following command to fetch all measurements, remember to replace "user" and "password"
+   with the real credentials:
+
+   ```sh
+   DATABASE_URL="postgres://user:password@localhost:5454/spark" node bin/fetch-historic-measurements.js <range-start> <range-end>
```

Comment:
```suggestion
   DATABASE_URL="postgres://user:password@localhost:5454/spark" node bin/fetch-historic-measurements.js [contract-address] <range-start> <range-end>
```

Based on the code below.

```js
if (args.length === 0 || !args[0].startsWith('0x')) {
  args.unshift(SparkImpactEvaluator.ADDRESS)
}
```

Reply:
25d0286